_projects/2018/astropy/sphinx-example-index.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Indexing examples in Sphinx docs
+desc: figure out a way to identify examples, tag them, and index them within Sphinx
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Knowledge of sphinx
+ - Interest in user training
+difficulty: moderate
+issues:
+# Related issues (if any) to this project.
+ - https://github.com/astropy/astropy/issues/6589
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Erik Tollerud
+ -
```

Comment:
potentially I'm happy to co-mentor this (pending on my involvement in other projects of course)